app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "title_page"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
Dùng toán tử 3 ngôi cho gọn code nha
```suggestion
    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```